pom.xml

```diff
@@ -160,7 +160,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-job</artifactId>
-      <version>2.26</version>
+      <version>2.32</version>
```

Comment:
or switch to BOM

Reply:
Indeed, switching this plugin to the BOM is definitely on my TODO list. I just haven’t gotten quite that far yet. Can look into it further as part of #35.

Reply:
Of course whatever you have time for is greta.